content/version/2/0/code-of-conduct.ko.md

```diff
@@ -0,0 +1,135 @@
++++
+version = "2.0"
+aliases = ["/version/2/0/ko"]
++++
+
+# 기여자 행동 강령 규약
+
+## 서약
+
+구성원, 기여자 및 리더로서 커뮤니티에서 연령, 신체 크기,
+눈에 보이거나 보이지 않는 장애, 민족성, 성별, 성 정체성과 표현,
+경력, 학력, 사회 경제적 지위, 국적, 외모, 인종, 종교
+또는 성적 정체성과 지향에 관계없이 모두에게 차별없이 참여할 것을 서약합니다.
+
+개방적이고 친근하며 다양하고 포용적이며 건강한 커뮤니티에 기여하는
+방식으로 행동하고 상호작용할 것을 서약합니다.
+
+## 표준
+
+커뮤니티의 긍정적인 환경을 위해 기여자가 해야 할 행동은 다음과 같습니다:
+
+* 소외하지 않고 배려하는 언어 사용
+* 서로 다른 경험과 관점 존중
+* 열린 마음으로 건설적인 비판을 수용
+* 커뮤니티에 가장 최선이 무엇인지에 주력
+* 다른 커뮤니티 구성원들에 대한 공감 표현
+
+* 다른 사람들에 대한 친절과 공감 표현
+* 다양한 의견, 관점 그리고 경험을 존중
```

Comment:
@jabbukka I cannot find this section from the English version. Maybe outdated contents :)

```suggestion
커뮤니티의 긍정적인 환경을 위해 기여자가 해야 할 행동은 다음과 같습니다:

* 다른 사람들에 대한 친절과 공감 표현
* 다양한 의견, 관점 그리고 경험을 존중
```
https://github.com/ContributorCovenant/contributor_covenant/blob/release/content/version/2/0/code_of_conduct.md
```
Examples of behavior that contributes to a positive environment for our
community include:

* Demonstrating empathy and kindness toward other people
* Being respectful of differing opinions, viewpoints, and experiences
* Giving and gracefully accepting constructive feedback
* Accepting responsibility and apologizing to those affected by our mistakes,
  and learning from the experience
* Focusing on what is best not just for us as individuals, but for the
  overall community
```

Reply:
Corrected! Thanks.